types/autobahn-browser/tslint.json

```diff
@@ -0,0 +1,78 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove all these rules and fix errors if any as new packages cannot have anything in their tslint otherthan extending the dtslint json file

Reply:
I've got a lot of errors after I removed all the rules. Most of them are `interface name must not have an "I" prefix`. That's how it was by default. I did not make it like that so not going to fix it.